packages/pluggableWidgets/switch-web/src/components/Alert.tsx

```diff
@@ -0,0 +1,10 @@
+import { FunctionComponent, createElement } from "react";
+
+export interface AlertProps {
+    message?: string;
+}
+
+export const Alert: FunctionComponent<AlertProps> = ({ message }) =>
```

Comment:
We already have this component in `piw-utils-internal` can you reuse it and remove this class?

Reply:
Thanks. Done.